src/fairseq2/recipes/lm/_online_finetune/_recipe.py

```diff
@@ -296,6 +296,11 @@ def load_online_finetuner(
         raise UnknownOnlineFinetuneUnitError(config.criterion.name) from None
 
     unit = unit_handler.create(model, gangs, config, vllm_actors)
+    try:
+        unit.maybe_sync_models(force_sync_vllm=True)
+    except AttributeError:
+        raise RuntimeError("Train unit does not support maybe_sync_models")
```

Comment:
sync vllm rollout model. note that this doesn't sync the ref model. they are usually not in step with each other